mongoquery/__init__.py

```diff
@@ -29,7 +29,10 @@ def _match(self, condition, entry):
                 for sub_operator, sub_condition in condition.items()
             )
         else:
-            if isinstance(entry, Sequence):
+            # strings are a subtype of Sequence, thus checking first
+            if isinstance(entry, string_type):
+                return condition == entry
+            elif isinstance(entry, Sequence):
```

Comment:
It could be cast as a compound conditional (e.g. if isinstance(Sequence) and not isinstance(string_type)).